src/components/change-requests/change-request-details/change-request-details/change-request-details.tsx

```diff
@@ -94,7 +103,11 @@ const ChangeRequestDetails: React.FC<ChangeRequestDetailsProps> = ({
         }
       />
       {buildDetails(changeRequest)}
-      <ReviewNotes reviewer={changeRequest.reviewer} reviewNotes={changeRequest.reviewNotes} />
+      <ReviewNotes
+        reviewer={changeRequest.reviewer}
+        reviewNotes={changeRequest.reviewNotes}
+        dateReviewed={changeRequest.dateReviewed!}
```

Comment:
It's usually better not to have not null assertions when they aren't necessary, here it would be better to have the `dateReviewed` be optional for ReviewNotes, just like `reviewer` and `reviewNotes`, following that all are optional in case it hasn't been reviewed yet.